src/gpu/generic/sycl/ref_matmul.cpp

```diff
@@ -52,16 +55,56 @@ void ref_matmul_t::pd_t::init_conf() {
     for (const auto &mdw : {src_d, weights_d, dst_d, bias_d}) {
         if (mdw.has_runtime_dims()) {
             any_runtime_params_ = true;
-            return;
+            return status::unimplemented;
         }
     }
-    init_rt_conf(conf_, src_d, weights_d, dst_d, bias_d);
+    init_rt_conf(conf_, data_md_t, dst_md_t, weights_md_t, src_d,
+            weights_d, dst_d, bias_d);
```

Comment:
```suggestion
    CHECK(init_rt_conf(conf_, data_md_t, dst_md_t, weights_md_t, src_d,
            weights_d, dst_d, bias_d));
```

Reply:
Check was not needed as currently function used to assert, now changed to return status and added check